doc/api/readline.markdown

```diff
@@ -41,6 +41,9 @@ the following values:
 
  - `historySize` - maximum number of history lines retained. Defaults to `30`.
 
+If a `completer` function is supplied, the user will not be able to enter tab
```

Comment:
I feel like this is sort of "well, duh". Perhaps it could be shortened and added into the `completer` arg description above? It seems like something one would expect rather than something that needs long documentation.

Thoughts?